src/main/java/thrift/logic/parser/ListCommandParser.java

```diff
@@ -39,10 +39,10 @@ private ListCommand getCommand(ArgumentMultimap argMultimap) throws ParseExcepti
             throw new ParseException(String.format(Messages.MESSAGE_INVALID_COMMAND_FORMAT,
                     ListCommand.MESSAGE_USAGE)); //prefix m/ exists, but there is invalid preamble before the prefix.
         } else if (arePrefixesPresent(argMultimap, CliSyntax.PREFIX_MONTH)) {
-            Month month = ParserUtil.parseMonth(argMultimap.getValue(CliSyntax.PREFIX_MONTH).get()); //to be used
-            return new ListCommand(); //filter by month tbc, to look like return new ListCommand(month);
+            Calendar date = ParserUtil.parseDate(argMultimap.getValue(CliSyntax.PREFIX_MONTH).get());
```

Comment:
Might want to use `getSingleValue(CliSyntax.PREFIX_MONTH)` since we must guarantee only one month is inputted?

Reply:
done so. nice change.